.github/workflows/tests.yml

```diff
@@ -20,17 +20,29 @@ jobs:
         laravel: [8, 9, 10, 11]
         exclude:
           - php: 7.3
-            laravel: [9, 10, 11]
+            laravel: 9
+          - php: 7.3
+            laravel: 10
+          - php: 7.3
+            laravel: 11
+          - php: 7.4
+            laravel: 9
           - php: 7.4
-            laravel: [9, 10, 11]
```

Comment:
GitHub Actions doesn't support providing array to `exclude` or `include` which cause the issue.